### ISSUE TITLE
Exception on Update-MgUser Disposed System.Net.Http.StringContent 

### ISSUE THREAD

**Issue by seanmcne** [microsoftgraph/msgraph-sdk-powershell#458]
Repro of the issue: 
``` powershell
Connect-Graph -Scopes "User.Read","User.ReadWrite.All","Mail.ReadWrite","Directory.Read.All","Chat.ReadWrite", "People.Read","Group.Read.All", "Tasks.ReadWrite","Sites.Manage.All"

$allUsers = Get-MgUser -All
$oldEmailAddressDomainUsers = allUsers|where {$_.mail -clike '*olddomain.com*'}|select Id,Mail,UserPrincipalName
$oldEmailAddressDomainUsers |%{
    $newMail = $_.Mail -replace "olddomain.com", "newdomain.com" 
    Write-Output "Updating $($_.Mail) to mail address of: $newMail"
    #this cmdlet is almost completely broken - but this is a way to trick it into working 
    #this cmdlet also produces an error but appears to ultimately complete the update
    Update-MgUser -UserId $_.Id -Mail $newMail -AdditionalProperties @{}  //<-- THROWS DISPOSED OBJECT EXCEPTION
}
```

![image](https://user-images.githubusercontent.com/2292260/98694014-97fbcd80-2336-11eb-95b7-0ff72616060d.png)

**Comment by peombwa** [comment]
@seanmcne which version of the SDK are you using? I'm not able to repro this with v1.1.0.
``` powershell
 Get-InstalledModule Microsoft.Graph.Users
```

**Comment by seanmcne** [comment]
![image](https://user-images.githubusercontent.com/2292260/98710218-fcc02380-2348-11eb-847d-7b54114b6740.png)

**Comment by seanmcne** [comment]
will reload and retest just to be sure shortly.

**Comment by seanmcne** [comment]
Ok, quite strange, I wonder if I had the old version still loaded prior to updating.  Must be fixed now in 1.1.0, once restarting the powershell process it seems to be working. Sorry for the false alarm!
